server/src/models/configModel.ts

```diff
@@ -1,5 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
+export type PostCallSurveyType = 'msforms' | 'onequestionpoll' | 'thirdparty';
```

Comment:
I'm thinking we shouldn't include `onequestionpoll` yet since it doesn't yet have a corresponding interface and it is not ready to be exposed.